.circleci/config.yml

```diff
@@ -243,6 +243,16 @@ jobs:
       - run:
           name: Set NPM authentication.
           command: echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
+      - run:
+          # This is needed because `npm publish` invokes `npm prepare`
+          # that invokes `npm run compile` to build the Javascript code.
+          # However, compilation cannot succeed if the build tools have
+          # not been installed.  Since the build tools are dev dependencies,
+          # NODE_ENV is set to `development`.
+          name: Install modules and dependencies.
+          command: npm install
+          environment:
+            NODE_ENV: development
```

Comment:
Interesting. Is this actually needed? All we do here is `npm install` and I don't think that depends on `NODE_ENV`.

Reply:
I've removed it.